blog/_posts/btcpay-server-2-1-0.md

```diff
@@ -0,0 +1,133 @@
+
+[![ECWID Integration for BTCPay Server](https://img.youtube.com/vi/5NOJBnQjsLc?/mqdefault.jpg)](https://www.youtube.com/watch?v=5NOJBnQjsLc)
+
+### Removing Watch-Only wallet creation by default
+
+To minimize user risks, v2.1.0 disables watch-only wallet creation for non-admin users by default. This decision addresses cases where users lost access to funds by failing to back up their seed phrases. Server administrators can still enable this feature, but users are encouraged to create watch-only wallets externally and connect them to BTCPay instead. This default behavior reduces support cases related to lost funds, promoting safer wallet management practices. Users who already have watch-only wallets, remain unaffected, as this feature only activates for new users.mFind more details [here](https://github.com/btcpayserver/btcpayserver/pull/6626).
+
+### Plugin store improvements
+
+Our plugin ecosystem is a cornerstone of our long-term vision, and we remain committed to improving the infrastructure around plugins while supporting developers in building innovative features.
+
+Over the past quarter, we introduced enhancements to the Plugin Builder’s administration, giving admins better tools to manage published plugins. These improvements lay the foundation for a smoother plugin release experience, which will continue to evolve in upcoming updates.
+
+As a first step toward strengthening the integrity and security of published plugins, we’ve introduced **email verification as a requirement before publishing**. Please note that this will be mandatory for all future plugin versions published on [https://plugin-builder.btcpayserver.org](https://plugin-builder.btcpayserver.org).
+
+The next step in this effort is GitHub account verification. While available today, it remains optional for now. In future versions, it will become a requirement. This phased rollout allows us to improve plugin authenticity without disrupting developer workflows.
+
+The next release of Plugin Builder will also include improvements to author verification, further streamlining the plugin publishing process and enhancing overall trust. We’re additionally working on **Featured Plugins**, which will highlight widely used and trusted plugins from reputable authors to help users discover the most reliable tools available.
+
+### Plugin Updates
+
+As part of our ongoing efforts to improve the plugin ecosystem, we're excited to highlight several new plugins and important updates to widely-used ones:
+
+* **Cash Checkout** – Adds a cash payment option to the checkout, ideal for physical merchants using the BTCPay POS app.  
+* [**Satoshi Tickets**](https://github.com/TChukwuleta/BTCPayServerPlugins/tree/main/Plugins/BTCPayServer.Plugins.SimpleTicketSales) – Event organizers, conference hosts, and community managers can now create and manage events directly from their BTCPay Server instance, with built-in ticketing.  
```

Comment:
@TChukwuleta @ndeet need to update this video once plugin rename is done.

Reply:
Done in https://github.com/btcpayserver/blog/pull/67/commits/72e3d3f2f4d004c3467b1ae38d99e839f43dca35